src/summersplash2022/nyutdannet/sections/hvorforAkkurattVariant.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react';
+import style from 'src/summersplash2022/index.module.css';
+import style2 from 'src/summersplash2022/nyutdannet/nyutdannet.module.css';
+import ReadMoreArrowWhite from '../img/lesMerPilHvit';
+import HandBooKPagesJob from '../utils/handBookPagesJob';
+
+const WhyVariant = () => {
+  const [whichButtonSelected, setWhichButtonSelected] = useState('');
+  const [buttonClicked, setButtonClicked] = useState(false);
+
+  const handleOnClick = (buttonValue: string) => {
+    if (window.matchMedia('(max-width: 900px)').matches) {
+      setButtonClicked(true);
+    }
+
+    setWhichButtonSelected(buttonValue);
+  };
+  return (
+    <section
+      className={style.whyWorkAtVariant}
+      id="hvorforjobbeivariant"
+      style={{
+        backgroundColor: '#01574F',
+        color: 'white',
+        height: buttonClicked
+          ? '1600px'
+          : '900px' || window.matchMedia('(min-width: 1550px)').matches
+          ? '1300px'
+          : '900px',
+      }}
+    >
+      <div className={style.handbook}>
+        <div className={style.handbookChildButton}>
+          <h3 className={style2.heading}>Hvorfor akkurat Variant?</h3>
+          <p className={style.positionTextLeft}>
+            Variant er en variant av et konsulentselskap som er raust, åpent og
+            læreglad. Disse verdiene ligger til grunn for hvordan vi møter
+            hverandre, våre kunder og alle andre. I håndboken vår kan du lese om
+            hvordan ting gjøres i Variant, hva vi prøver å oppnå og hvorfor vi
+            tenker som vi gjør. Under kan du sjekke ut noen utvalgte temaer.
+          </p>
+
+          <div className={style.handbookGrid}>
+            <button
+              style={{ color: 'white' }}
+              className={style.handbookButton1}
+              onClick={() => handleOnClick('Formal og verdier')}
+            >
+              Formål og verdier
+            </button>
+            <button
+              style={{ color: 'white' }}
```

Comment:
Disse lenkene får litt feil kontraster. 

![Screenshot 2022-08-29 at 11 19 30](https://user-images.githubusercontent.com/606374/187168447-460b5956-4675-42ff-bca5-700f02641347.png)


Styleguide er lagt opp slik at tekstfarge skal matche for rett kontrastnivå:
![Screenshot 2022-08-29 at 11 20 10](https://user-images.githubusercontent.com/606374/187168708-99846d2e-c8c4-4a12-ab35-ca42bd0337b1.png)

Skal også kunne bruke fargene direkte via CSS slik:

```css
.myRedButton {
  background-color: var(--color-primary__shade1);
  color: var(--color-primary__shade1--text);
}
```

Formatet her er altså `--color-<navn_på_farge>--text` og du vil få automatisk rett kontrast på teksten

Reply:
For spesielt interesserte: Kontrasten jeg snakker om her altså er lover og regler for universell utforming. Det er kontrastforholdstall som er kalkulert for å kunne være synlig for folk med nedsatt syn og generelt for tydelig visning av tekst. Kan lese mer om det her https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

Vi er ikke underlagt de lovene og reglene og kan ikke bøtlegges da vi ikke er en offentlig tjeneste eller offentlig selskap, men det er god praksis å følge de og selvfølgelig bra for de som har behov for denne kontrasten (som kan være alle slags mulige folk)